plugins/development/plugin-git/src/node/gitPlugin.ts

```diff
@@ -60,25 +41,63 @@ export const gitPlugin =
           return
         }
 
+        if (filter && !filter(page)) return
+
+        const { frontmatter } = page
+
+        if (
+          !(frontmatter.changelogs ?? contributors) &&
+          !(frontmatter.contributors ?? changelogs) &&
+          !createdTime &&
+          !updatedTime
```

Comment:
I am not understanding this check at first sight, and the second line seems weird. Some comments should be added here.